camunda-formio-tasklist-vue/src/styles/camundaFormIOFormList.scss

```diff
@@ -47,4 +47,18 @@ font-family: Nunito Sans, SemiBold;
 display: flex;
 justify-content: center;
 margin-top: 15px;
-}
\ No newline at end of file
+}
+
+//TODO Remove this when Semantic UI css is removed. Semantic UI css override formio css tooltip
+
+[data-tooltip]:before {
+  visibility: unset !important;
+  position: initial !important;
```

Comment:
the same logic is also required in the Tasklist view. Is this file only for the FormList modal?